server/command.go

```diff
@@ -250,6 +254,64 @@ func executeInstanceDelete(p *Plugin, c *plugin.Context, args ...string) *model.
 	return executeInstanceSelect(p, c, "1")
 }
 
+func (p *Plugin) transitionJiraIssue(mmUserId, issueKey, toState string) error {
```

Comment:
1/5 `issue.go`? (not a blocking issue though)

Reply:
Oh ya, I totally meant to do that. Fixed.
Unrelated, we really need to separate our concerns api vs app logic like transitionJiraIssue.